### PR TITLE
Reboot cron job

### PR DESCRIPTION
**Pull request by Dashboy1998**
## Context 

* Adds a cron job to automatically reboot the server as mentioned in #259 

## Choices

* Already using cron for automatic updates and backups

## Test instructions

1. Enable rcon & auto reboots 
2. Set the reboot expression near your current time so you don't wait long
3. Start the container
4. Join the game and wait for the reboot

image

## Checklist before requesting a review

* [x] I have performed a self-review of my code
* [x] I've added documentation about this change to the README.
* [x] I've not introduced breaking changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,6 +293,30 @@ AUTO_UPDATE_CRON_EXPRESSION is a cron expression, in a Cron-Expression you defin
...
+
+Set AUTO_REBOOT_ENABLED enable or disable automatic backups (Default is disabled)
+
+AUTO_REBOOT_CRON_EXPRESSION is a cron expression, in a Cron-Expression you define an interval for when to run jobs.
+
+> [!TIP]
+> This image uses Supercronic for crons
+> see [supercronic](https://github.com/aptible/supercronic#crontab-format)
+> or
+> [Crontab Generat](https://crontab-generator.org).
```
Comment (Technetium1) [inline README.md:315]: @Dashboy1998 Should be `Generator`

**Comment by Technetium1** [comment]
Could be nice to add in https://github.com/thijsvanloef/palworld-server-docker/blob/main/.env.example

**Comment by MSpreckels** [comment]
Dont we already have a reboot cronjob? #206

**Comment by Dashboy1998** [comment]
> Dont we already have a reboot cronjob? #206

That appears to be specifically for Kubernetes whereas this will work for the stand alone container.

**Comment by MSpreckels** [comment]
You are correct. Does it make sense to keep the k8s only variant of a daily reboot when it will be implemented in the container?

**Comment by Dashboy1998** [comment]
> You are correct. Does it make sense to keep the k8s only variant of a daily reboot when it will be implemented in the container?

I think the k8s version has some specific benefits for k8s. I don't use k8s so I'm not really sure of the benefits of having a separate solution for them.

**Review by thijsvanloef — APPROVED** [review]
LGTM

**Comment by Technetium1** [comment]
@MSpreckels keeping both makes sense to me because they are not restarting _exactly_ the same way. There's more granularity available with the Kubernetes spec, such as [killing the job if it times out](https://github.com/thijsvanloef/palworld-server-docker/blob/24c4071ba1fa690ffe98011680ea680f1ef247e7/k8s/examples/cronjob-reboot/cronjob.yaml#L11), and the fact that prevents concurrent jobs from executing at once, but the same effect is unnecessary with only 1 Docker container. Even moreso because there's potential for that job to run on multiple replicas in a cluster (even if they don't exist right now); with offset times, or just waiting for completion on each replica.

Worth noting that `AUTO_REBOOT_ENABLED=false` to prevent issue for k8s users.

**Pull request merged**